context/metadata.ts

```diff
@@ -16,6 +17,17 @@ const dispatch: { [key: string]: version } = {
   "0.1.1": "0.1.1",
 };
 
+// Those values are from tzkt api: /v1/contracts
+type typeHash = string;
+type codeHash = string;
+
+// Before 0.0.10, the version is stored on the contract so tzip16 won't failed to retrieve it
+const VERSION_HASH: { [k: `${typeHash}:${codeHash}`]: version } = {
+  "-357299388:-2016479992": "0.0.10",
+  "-483287042:793087855": "0.0.11",
```

Comment:
It's before `0.0.11`, no need for `0.0.10`
I think the `codeHash` of `0.0.11` is `521053333`.

```suggestion
// Before 0.0.11, the version is stored on the contract so tzip16 won't failed to retrieve it
// typeHash and codeHash are provided by tzkt API
const VERSION_HASH: { [k: `${typeHash}:${codeHash}`]: version } = {
  "-483287042:521053333": "0.0.11",
```